app/src/main/java/com/concordia/mcga/activities/SplashActivity.java

```diff
@@ -0,0 +1,37 @@
+package com.concordia.mcga.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+
+
+public class SplashActivity extends AppCompatActivity {
+
+    private final long VIEW_TIME = 2000;
+
+    /**
+     * Creates the splash screen. Switch to MainActivity after 'VIEW_TIME' milliseconds
+     * @param savedInstanceState The savedInstanceState is a reference to a Bundle object that is passed into the onCreate method of every Android Activity
+     */
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.splash_screen);
```

Comment:
The logo looks too big IMHO. It would look nicer if it was rendered at centred at 1/3 the width of the screen...